automation/src/main/java/org/wso2/carbon/testgrid/automation/beans/TestNGTest.java

```diff
@@ -19,15 +19,39 @@
 package org.wso2.carbon.testgrid.automation.beans;
 
 import org.wso2.carbon.testgrid.automation.exceptions.TestGridExecuteException;
+import org.wso2.carbon.testgrid.automation.executers.common.TestExecuter;
+import org.wso2.carbon.testgrid.automation.executers.common.TestExecuterFactory;
 import org.wso2.carbon.testgrid.common.Deployment;
+import org.wso2.carbon.testgrid.common.constants.TestGridConstants;
+
+import java.util.List;
 
 /**
  * This class is responsible for Executing the TestNG tests.
  */
 public class TestNGTest extends Test {
 
+    private List<String> testNGJars;
+    private TestExecuter testExecuter = TestExecuterFactory.getTestExecutor(TestGridConstants.TEST_TYPE_TESTNG);
+
+    private List<String> getTestNGJars() {
+        return testNGJars;
+    }
+
+    public void setTestNGJars(List<String> testNGJars) {
```

Comment:
The String here correspond to a path in a file system, right? How about we accept a List of java.nio.file.Path instances?

Reply:
This causes interface level changes. Thus created the issue #53 for this.